TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Owner: {Your Name} {you@example.com}, {@you} on the Elm Slack
```

Comment:
"point person" also conveys it well but it's awkward and people to gender it unnecessarily.